android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -329,7 +334,7 @@ public void processProperties(KrollDict d)
 		TiUIHelper.linkifyIfEnabled(tv, d.get(TiC.PROPERTY_AUTO_LINK));
 		tv.invalidate();
 	}
-	
+
```

Comment:
Yes, I did the backport manually which is linked [here](https://github.com/appcelerator/titanium_mobile/pull/8885).